tests/sentry/integrations/slack/test_notify_action.py

```diff
@@ -89,7 +89,7 @@ def test_render_label_without_integration(self):
         label = rule.render_label()
         assert (
             label
-            == "Send a notification to the [removed] Slack workspace to #my-channel (optionally, an ID: ) and show tags [] in notification"
+            == "Send a notification to the [removed] Slack workspace to #my-channel (optionally, an ID: )"
```

Comment:
Test that ` and show tags [] in notification` is no longer included if `tags = ""`